config.template.yaml

```diff
@@ -4,11 +4,11 @@
 # user: username to authenticate as
 # password: password for user
 mysql:
-  server: 'localhost'
-  port: 3306
-  database: 'cytube3'
-  user: 'cytube3'
-  password: ''
+  server: '${MYSQL_HOST}'
```

Comment:
Is there any way to make Docker use it's own separate config template? These changes make it much more annoying for regular humans to edit the template to working order